packages/fluentui/react-northstar/test/specs/components/Provider/Provider-test.tsx

```diff
@@ -287,4 +289,42 @@ describe('Provider', () => {
       expect(removeEventListener).toHaveBeenCalledTimes(5);
     });
   });
+
+  describe('document.body', () => {
+    it('adds an element to document.body', () => {
+      const className = faker.lorem.word();
+      const wrapper = mount(
+        <Provider className={className}>
+          <div />
+        </Provider>,
+      );
+
+      expect(document.querySelector(`.${className}`)).toBeInstanceOf(Element);
```

Comment:
nit: if you are checking if an element is / is not in the document, using `(.not).toBeInTheDocument()` might make the assertion easier to understand.

Reply:
Yeah, this looks much cleaner. Thanks for the suggestion